### PR TITLE
feat: add cdn conf

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)

@raisedadead this is modelled on the devdocs config, albeit simplified a bit. I'd like to test this out on a staging instance before deploying for real, but I think it's basically fine.

### CODE REVIEW

File: sites-enabled/40-cdn.freecodecamp.org.conf [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  server_name cdn.freecodecamp.org;
+
+  return 301 https://cdn.freecodecamp.org$request_uri;
+}
+
+# health check
+server {
+  listen 64555;
+  return 200;
+}
```
Comment (raisedadead) [inline sites-enabled/40-cdn.freecodecamp.org.conf:32]:
```suggestion
}

```

Nitpick: Insert newline here. NGINX configs are packaged into a giant single file.

**Comment by raisedadead** [comment]
Can I take a look at the devdocs configs for comparison?

**Comment by ojeytonwilliams** [comment]
Sure, https://github.com/freeCodeCamp/devdocs-nginx-config is what's currently live on the proxies.

**Comment by raisedadead** [comment]
Hi @ojeytonwilliams 

While I am happy with these changes. I think as discussed; we can use CF to S3 Mapping directly. If you are interested please go ahead and tighten the bucket permissions as you see fit like you mentioned during the call.

**Pull request closed without merging**